application/job/job.go

```diff
@@ -42,11 +42,11 @@ func newThreadMessageHandler(args worker.Args) error {
 	}
 
 	var m models.Message
-	if err := m.FindByID(id, "SentBy", "Thread"); err != nil {
+	if err := m.FindByID(models.DB, id, "SentBy", "Thread"); err != nil {
```

Comment:
Lots of `models.DB` in this file, is that right?

Reply:
I'm thinking so if this runs as a background job and does not have a user context to pull a Tx from.

Reply:
Right